packages/node_modules/@webex/widget-voice-mail/src/VoicemailPlaybackControls.styles.scss

```diff
@@ -8,7 +8,8 @@ $C: wxc-voicemail-playback-controls;
 
 .vmLoader .md-loading-spinner-wrapper {
   position: relative;
-  width: 28px;
+  width: 23px;
```

Comment:
Can we check with rem values instead of px

Reply:
In the next PR will be changing px to  rem